user-manuals/overworld/README.md

```diff
@@ -3,28 +3,37 @@
 The overworld is the core of the Gamify-IT platform.
 It is the main interface for students to interact with the platform.
 
-The overworld consists of multiple regions and dungeons.
-Throughout the world, students can discover course contents in form of [minigames](../README.md) and NPC dialogues.
+## Overview
+
+The overworld consists of multiple `worlds` and `dungeons`.
+Throughout these areas students can discover course contents like playable [minigames](../README.md) and dialogues with `NPCs`. The player (the character in the middle of the picture) can walk around and interact with various objects such as the NPC on the left or `books` providing useful information. The `HUD` helps navigating with the minimap and keeping track of the progress with the progress bar.  
 ![overworld](../../images/overworld.webp)
 
-- The player is in the center of the screenshot
-- They can walk around and interact with various objects such as the NPC on the left.
-- There is a HUD which displays a minimap, and other useful information
+## Gameplay
+
+### NPCs
 
+The player can talk to other characters in the world to obtain information. The lecturer creating the course can specify what each `NPC` has to say and therefor give tips and summaries of important topics to the player.
```

Comment:
```suggestion
The player can talk to other characters in the world to obtain information.  
The lecturer can specify what each `NPC` has to say.  
This can be used to give tips or to summarize important topics for the player.
```